russstation/code/modules/mob/living/carbon/human/species_types/diona.dm

```diff
@@ -30,60 +47,47 @@
 
 	return randname
 
-/datum/species/diona/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
-	C.draw_russ_parts()  //changes icons to be fetched from russstation/icons/mob/mutant_bodyparts.dmi
-	. = ..()
-	C.faction |= "plants"
-	C.faction |= "vines"
-
-/datum/species/diona/on_species_loss(mob/living/carbon/C, datum/species/old_species, pref_load)
-	C.draw_russ_parts(TRUE)  //icon path is reset to default
-	. = ..()
-	C.faction -= "plants"
-	C.faction -= "vines"
-
-/datum/species/diona/spec_life(mob/living/carbon/human/H)
+// Similar to podpeople.dm
+/datum/species/diona/spec_life(mob/living/carbon/human/H, delta_time, times_fired)
 	if(H.stat == DEAD)
 		return
+
 	var/light_amount = 0 //how much light there is in the place, affects receiving nutrition and healing
 	if(isturf(H.loc)) //else, there's considered to be no light
 		var/turf/T = H.loc
-		light_amount = min(1,T.get_lumcount()) - 0.5
-		H.adjust_nutrition(light_amount * 10)
+		light_amount = min(1, T.get_lumcount()) - 0.5
+		// Maybe set to `5 *` like podpeople?
+		H.adjust_nutrition(10 * light_amount * delta_time)
 		if(H.nutrition > NUTRITION_LEVEL_ALMOST_FULL)
 			H.set_nutrition(NUTRITION_LEVEL_ALMOST_FULL)
 		if(light_amount > 0.2) //if there's enough light, heal
-			H.heal_overall_damage(2,1, 0, BODYTYPE_ORGANIC)
+			// OLD HEAL CODE
+			// H.heal_overall_damage(2,1, 0, BODYTYPE_ORGANIC)
+			// New Code | Might need to tweak numbers to match podpeople `0.5 * delta_time, 0.5 * delta_time`
+			H.heal_overall_damage(2 * delta_time, 1 * delta_time, 0, BODYTYPE_ORGANIC)
+			H.adjustToxLoss(-0.5 * delta_time)
```

Comment:
Would this be a buff to their healing by adding extra toxin healing under light?

Reply:
It's possible it's a buff, as well as the new heal uses a delta which I assume means previously it could miss updates so that might be increased as well.

When I mentioned it on discord it was mentioned them being so slow and being not played directly meant it likely wouldn't be an issue. The ToxLoss addition was based off podpeople; but they have lower heal amount (noted in the comment).

I can't speak to balance personally but can easily tweak these numbers if needed.

Reply:
I suppose it makes sense to buff them a small bit considering how little play they got